subt/docker/robotika/ros/robotika/launch/robot.launch

```diff
@@ -0,0 +1,287 @@
+  </node-->
+  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find robotika)/../robotika.rviz" />
+  <!-- NAVIGATION -->
+  <node pkg="move_base" type="move_base" respawn="false" name="move_base">
+	  <rosparam file="$(find robotika)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
+	  <rosparam file="$(find robotika)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
+	  <rosparam file="$(find robotika)/params/move_base_params.yaml" command="load" />
+	  <rosparam file="$(find robotika)/params/local_costmap_params.yaml" command="load" />
+          <rosparam file="$(find robotika)/params/global_costmap_params.yaml" command="load" />
+          <rosparam file="$(find robotika)/params/base_local_planner_params.yaml" command="load" />
+	  <param name="controller_frequency" value="10.0"/>
+	  <remap from="cmd_vel" to="$(arg robot_name)/cmd_vel" />
+	  <!--remap from="map" to="/rtabmap/proj_map" /-->
+	  <remap from="map" to="/rtabmap/grid_map" />
+	  <remap from="odom" to="$(arg robot_name)/odom" />
+	  <param name="DWAPlannerROS/global_frame_id" value="$(arg robot_name)/odom"/>
+	  <param name="DWAPlannerROS/odom_topic" value="/$(arg robot_name)/odom"/>
+	  <param name="DWAPlannerROS/map_frame" value="$(arg robot_name)/odom"/>
+	  <param name="global_costmap/robot_base_frame" value="$(arg robot_name)/base_link"/>
+	  <param name="global_costmap/global_frame" value="$(arg robot_name)/odom"/>
+	  <param name="local_costmap/global_frame" value="$(arg robot_name)/odom"/>
+	  <param name="local_costmap/obstacles/laser_scan_sensor/sensor_frame" value="$(arg robot_name)/base_link/camera_front"/>
+	  <param name="local_costmap/obstacles/laser_scan_sensor/topic" value="/scan"/>
+	  <param name="local_costmap/robot_base_frame" value="$(arg robot_name)/base_link"/>
+	  <param name="global_costmap/robot_base_frame" value="$(arg robot_name)/base_link"/>
+  </node>
+
+<!-- REMOTE CONTROL /-->	
+	<!--node name="keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
+		<param name="~speed" value="0.4" />
+		<param name="~turn" value="1.0" />
+		<remap from="cmd_vel" to="/$(arg robot_name)/cmd_vel"/>
+	</node-->
+
+
+	<!--CAMERA FRAME_ID REMAPING BECAUSE OF BUG IN SIMULATOR-->
+  	<node name="frame_changer_front_depth" pkg="robotika" type="image_frame_changer.py">
+          	<param name="src_topic" value="/$(arg robot_name)/front/depth"/>
+          	<param name="dest_topic" value="/$(arg robot_name)/front/depth_optical"/>
+	  	<param name="dest_frame" value="$(arg robot_name)/base_link/camera_front_optical"/>
+	</node>
+	<node name="frame_changer_front_rgb" pkg="robotika" type="image_frame_changer.py">
+		<param name="src_topic" value="/$(arg robot_name)/front/image_raw"/>
+		<param name="dest_topic" value="/$(arg robot_name)/front/image_raw_optical"/>
+          	<param name="dest_frame" value="$(arg robot_name)/base_link/camera_front_optical"/>
+	</node>
```

Comment:
I'd like to know more about this. What is the problem? What is the proposed solution?

Reply:
It is a solution for this: 
https://github.com/osrf/subt/issues/369

https://osrf-migration.github.io/subt-gh-pages/#!/osrf/subt/issues/184/page/1?slug=urban-feature-camera-coordinate-frame